scripts/README.md

```diff
@@ -0,0 +1,12 @@
+Instructions for esmDependencyCheck.js
```

Comment:
can we have that as an h2? e.g. `## Instructions for... `

Reply:
I have now learned that this is a thing that can be done. Thanks